lms/templates/provider/authorize.html

```diff
@@ -1,59 +1,66 @@
-{% extends "main_django.html" %}
+## mako
```

Comment:
How does this template get rendered? I can't find any references to authorize.html in edx-platform or the configuration repo.